types/natural-compare/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for natural-compare 1.4
+// Project: https://github.com/litejs/natural-compare-lite
+// Definitions by: Doniyor Aliyev <https://github.com/doniyor2109>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function naturalCompare(a: string, b: string): -1 | 0 | 1;
```

Comment:
Is this supposed to be always identical to `natural-compare-lite`? 

I wonder if it would be a good idea to prevent divergence by just doing a re-export like:

```ts
import nc = require('natural-compare-lite');
export = nc;
```

I'm not familiar with the packages so feel free to reject the suggestion.

Reply:
I _think_ so I'm not too familiar with it either. However, the modules are unchanged for 4 years, so risk of divergence should be low

Reply:
Sounds fine to me. Looks pretty simple and explicit as is.